docs/Displaying a User Interface/Template Images/index.md

```diff
@@ -8,6 +8,36 @@ You can easily display text, images, and buttons using the @![iOS]`SDLScreenMana
 | secondaryGraphic | The second image in a template that supports multiple images |
 
 ## Showing Images
+### Creating an Image to Upload
+Create an `SDLArtwork` object which can be manually uploaded or set into the @![iOS] `SDLScreenManager`!@ @![android, javaSE, javaEE] `ScreenManager`!@ and automatically uploaded. It includes information about whether the image should be persisted between vehicle startups, whether the image is a template image and should be re-colored, and more.
+
+@![iOS]
+|~
+```objc
+<#Stub#>
+```
+```swift 
+let <#your variable name here#> = SDLArtwork(image: UIImage(named: <#Name to Upload As#>)!, persistent: true, as: .PNG /* or .JPG */)
+```
+~|
+!@
+
+@![android, javaSE, javaEE]
+```java
+
+<#Stub#>
+```
+!@
+
+@![javascript]
+```js
+<#Stub#>
+```
+!@
+
+
+### Setting Primary Graphic 
+
```

Comment:
```suggestion
```

Reply:
What are we suggesting here?  Should we drop this header?  Or perhaps change it to be more accurate and flow better with the section?

Reply:
This extra line hasn't been removed yet.

Reply:
Still not removed